src/components/UrlFilter/index.jsx

```diff
@@ -0,0 +1,262 @@
+          <div className="mr-1">{urls.length}</div>
+          <div>results</div>
+        </div>
+      </div>
+      <div className="h-240 w-[1400px] bg-white border border-gray-400">
+        <div className="h-16 flex items-center flex-row bg-gray-100 text-xl border-b border-gray-500 mb-2 rounded-lg">
+          <div className="flex items-center justify-center w-[100px] border-r border-gray-500">
+            <p>TREND</p>
+          </div>
+          <div className="flex items-center justify-center w-[560px] border-r border-gray-500">
+            <p>LONG LINK</p>
+          </div>
+          <div className="flex items-center justify-center w-[180px] border-r border-gray-500">
+            <p>ORGANIZATION</p>
+          </div>
+          <div className="flex items-center justify-center w-[180px] border-r border-gray-500">
+            <p>CREATED AT</p>
+          </div>
+          <div className="flex items-center justify-center w-[180px] border-r border-gray-500">
+            <p>TOTAL CLICKS</p>
+          </div>
+          <div className="flex items-center justify-center w-[200px]">
+            <p>ACTIONS</p>
+          </div>
+        </div>
+        <div className="flex aligns-center flex-col px-[8px]">
+          {urls.map((url) => (
+            <div className=" h-14 w-full flex aligns-center flex-row border border-gray-100 mb-2 rounded-lg text-base">
+              <div className="flex items-center justify-center w-[92px]">
+                <img
+                  src={GraphIcon}
+                  fill="#0f9d58"
+                  className="w-7 h-7 mr-2"
+                  alt="graph icon increase"
+                />
+                <p className="text-green-700">2</p>
+              </div>
+              <div className="h-14 w-[520px] max-w-[520px] flex items-center text-left mr-[40px] truncate">
+                {url.link}
+              </div>
+              <p className="inline-flex items-center text-left w-[160px] mr-[20px] truncate">
+                {url.org}
+              </p>
+              <div className="inline-flex items-center justify-center w-[180px]">
+                {url.date}
+              </div>
+              <div className="flex items-center justify-center w-[180px]">
+                {url.totalClicks}
+              </div>
+              <div className="flex items-center justify-center flex-row w-[192px]">
+                <button
+                  type="button"
+                  className="w-7 h-7 flex items-center justify-center cursor-pointer mr-4 bg-gray-300 rounded-md"
+                >
+                  <img
+                    src={AddToBlackList}
+                    alt="add-to-blacklist"
+                    className="w-5 h-5"
+                    fill="#4285F4"
+                    opacity="0.87"
+                  />
+                </button>
+                <button
+                  type="button"
+                  onClick={() => DeleteUrl(url)}
```

Comment:
tên hàm, tên biến mình đặt theo camelCase nhé em, đọc lại naming convention nha
```suggestion
                  onClick={() => deleteUrl(url)}
```